client/web/src/Routes/Routes.jsx

```diff
@@ -22,6 +23,9 @@ const Route = () => {
           <PublicRoute path="/login">
             <Login />
           </PublicRoute>
+          <AdminRoute exact path="/api/question" >
+            <Questions />
```

Comment:
Don't add API in the frontend path

Reply:
There is already a route called questions for the private route, so pull the latest and make changes to admin route question